cpp/dolfinx/fem/interpolate.h

```diff
@@ -278,7 +309,9 @@ void interpolate_c(
       x.rows(), value_size);
   f(values, x);
 
-  detail::interpolate_values<T>(u, values);
+  // FIXME: This looks strange. Check and add comments
```

Comment:
Seems like an appropriate thing to check

Reply:
@garth-wells Did you leave this FIXME?

Reply:
I thought this go removed. I have another branch where it will get cleaned up.

Reply:
This code path is untested as far as I can tell. Should we add a minimal test (interpolation of CG N <-> CG 1 for instance)?